src/ansi-c/gcc_version.cpp

```diff
@@ -160,4 +162,7 @@ void configure_gcc(const gcc_versiont &gcc_version)
   config.ansi_c.gcc__float128_type =
     gcc_version.flavor == gcc_versiont::flavort::GCC &&
     gcc_version.is_at_least(4u, gcc_float128_minor_version);
+
+  config.ansi_c.c_standard = gcc_version.default_c_standard;
+  config.cpp.cpp_standard = gcc_version.default_cxx_standard;
 }
```

Comment:
This is a bigger change in behavior; it may make sense to do so, but I'd take it out for now.